### DISCUSSION TITLE
Expose multiple packages as a single package

### DISCUSSION THREAD

**Discussion by Jighdan** [vercel/turborepo#2222] [Help] [answered]
Let's consider I have the following structure in my mono repo:

```
.
├── package.json
├── configs
        ├── stylelint
        |  ├── package.json
        |  ├── config.js
        |  └── index.js
        └── prettier
           ├── package.json
           ├── config.json
           └── index.js
└── boilerplates
        └── next
           ├── package.json
           ├── tsconfig.json
           └── index.ts
```

I'm trying to "expose" `configs/stylelint` and `configs/prettier` on `boilerplates/next` - the idea is to have multiple standalone configurations as packages and be able to "bundle" them together for their common use case as boilerplates - how can this be done?

These packages

**Comment by anthonyshew** [comment, marked as answer]
You could create an intermediary package that re-exports all of the bits you're interested in abstracting.
